README.md

```diff
@@ -33,7 +39,8 @@
 
 To get a local copy up and running:
 
-You should have basic knowledge of HTML, Webpack, Javascript, a text editor and chrome browser.
+You should have basic knowledge of HTML, Webpack, Javascript, APIs, a text editor and chrome browser.
+This project uses API to get and receive data, for detailed information on how to use this API, check out the [Leaderboard API documentation](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
```

Comment:
- [ ] According to the last instruction I'm sure you could be more specific about how to use the API, maybe you can include the link, how you can create a game and then the link to see the score list, and how to create a new score, I kindly invite you to be more specific about all these details.
- What exactly needs to include:
- README file looks professional and contains all that's needed for running the app with the API(s) it consumes. `It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary)`.